packages/client/src/Rest.ts

```diff
@@ -72,7 +73,8 @@ export class Rest implements Context {
         )
     }
 
-    post<T extends object>(urlParts: UrlParts, body: any, options: FetchOptions) {
+    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+    post<T extends object>(urlParts: UrlParts, body: any, options: FetchOptions): Promise<T> {
```

Comment:
```suggestion
    post<T extends object>(urlParts: UrlParts, body: object, options: FetchOptions): Promise<T> {
```
Maybe we could use https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-2.html#object-type?

Reply:
FIxed